sample_files/.travis.yml

```diff
@@ -14,8 +14,10 @@ env:
   - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
   - TRANSIFEX_USER='transbot@odoo-community.org'
   # This line contains the encrypted transifex password
-  # To encrypt transifex password, use
-  # $ travis encrypt TRANSIFEX_PASSWORD=your-password -r owner/project
+  # To encrypt transifex password, install travis ruby utils with:
+  #   $ gem install travis --user-install
+  # and use:
+  #   $ travis encrypt TRANSIFEX_PASSWORD=your-password -r owner/project
```

Comment:
@pedrobaeza I'm not talking about the OCA's transbot user, but about the token for transifex (`your-password` here).